peercrawler.py

```diff
@@ -275,22 +275,22 @@ def register_connections(self, peer: Peer, new_peers: Iterable[Peer]):
                 peer_connections.remove(new_peer)
             peer_connections.add(new_peer)
 
-    def get_connections(self):
-        return self.__connections.copy()
+    def get_connections(self) -> dict[Peer, set[Peer]]:
+        return copy.deepcopy(self.__connections)
 
     def run(self, initial_peer: Peer, interval_seconds=5):
-        initial_peers = self.peerman.get_peers_from_peer(initial_peer)
-        self.register_connections(initial_peer, initial_peers)
+        self.__connections[initial_peer] = set()
 
         while True:
             peers_list = [peer for peer in self.__connections]
             for peer in peers_list:
                 new_peers = self.peerman.get_peers_from_peer(peer)
                 self.register_connections(peer, new_peers)
-                print(f"Received peers from {peer.ip}, active peer count is {self.__connections[peer].__len__()}")
+                print(f"Received peers from {peer.ip}, active peer count for this peer is now {self.__connections[peer].__len__()}\nNow tracking {len(self.__connections)} peers in total")
```

Comment:
This is a bit unreadable, can you break to 2 different print statements?
Efficiency doesn't matter, readability is more important.